core/src/main/scala/org/apache/spark/rpc/netty/Dispatcher.scala

```diff
@@ -147,13 +147,15 @@ private[netty] class Dispatcher(nettyEnv: NettyRpcEnv, numUsableCores: Int) exte
   /** Posts a one-way message. */
   def postOneWayMessage(message: RequestMessage): Unit = {
     postMessage(message.receiver.name, OneWayMessage(message.senderAddress, message.content),
-      (e) => e match {
+      {
         // SPARK-31922: in local cluster mode, there's always a RpcEnvStoppedException when
         // stop is called due to some asynchronous message handling. We catch the exception
         // and log it at debug level to avoid verbose error message when user stop a local
         // cluster in spark shell.
         case re: RpcEnvStoppedException => logDebug(s"Message $message dropped. ${re.getMessage}")
-        case _ => throw e
+        case e if SparkEnv.get.isStopped =>
+          logDebug(s"Message $message dropped due to sparkEnv is stopped. ${e.getMessage}")
```

Comment:
I might log this as a warning at least, but looks OK otherwise

Reply:
OK

Reply:
@sqlwindspeaker do you want to update this?

Reply:
yes, I'll update this today, sorry for the delay

Reply:
> @sqlwindspeaker do you want to update this?

updated